janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java

```diff
@@ -349,6 +350,10 @@ CqlSession initializeSession() throws PermanentBackendException {
 
         configureRequestTracker(configuration, configLoaderBuilder);
 
+        if(configuration.has(SESSION_LEAK_THRESHOLD)){
+            configLoaderBuilder.withInt(DefaultDriverOption.SESSION_LEAK_THRESHOLD, configuration.get(SESSION_LEAK_THRESHOLD));
```

Comment:
I don't know why codecov doesn't see coverage of this test. I checked `shouldCreateCQLSessionWithDisabledSessionLeakThreshold` test locally (the one which is added in this PR) and it clearly goes here and the test passes.

Reply:
I can confirm it too. Moreover, CQLConfigTest had 15 test cases executed on master https://github.com/JanusGraph/janusgraph/runs/3340329250#step:6:92, but 16 test cases executed on this PR: https://github.com/JanusGraph/janusgraph/pull/2758/checks?check_run_id=3330352401#step:6:96, which implies this the `shouldCreateCQLSessionWithDisabledSessionLeakThreshold` is executed.